# Funciones para modelar la relación entre variables

# Clase 3

## Función polinomial

Recordemos que un polinomio es la suma de términos de que tiene una variable elevada a una potencia entera positiva.

$$ x^5-6x^4+x^2-10 $$

Una función polinomial tiene la forma

$$ f(x)=ax^4+bx^3+cx^2+dx+e $$

El grado de la función polinomial es el máximo el valor más grande sobre el exponente de la variable independiente.

El dominio máximo de cualquier polinomio es $\mathbb{R}$.

### Función lineal (de primer grado)

$$ f(x)=ax+b $$

**¿A qué se parece?** A la ecuación de una recta.

* ***Ejemplo:*** $f(x)=-4x+7$

* ***Ejemplo:*** $g(t)=\dfrac{3}{2}-\dfrac{5}{2}t$

El dominio máximo de cualquier función lineal es $\mathbb{R}$.

El rango para el dominio máximo de cualquier función lineal es $\mathbb{R}$.

## Dominio natural

*Son los valores de la variable independiente para los que la función está bien definida y tienen sentido.*

* ***Ejemplo:*** La función 

$$\rm{Chamba}\,(t)=1-\dfrac{1}{24}t$$ 

nos dice que tan eficientes somos al chambear en un día entero. Si la variable independiente $t$ representa el tiempo que ha pasado al comenzar un día en horas, ¿cuál es su dominio máximo y su dominio natural?

$\text{Dominio máximo}=\mathbb{R}=(-\infty,\infty)$

$\text{Dominio natural}=[0,24]$

* ***Ejemplo:*** Cuadrado del examen 1.

La diagonal de un cuadrado mide $x$ unidades. Su área, en unidades cuadradas, es

$$ A(x)=\dfrac{x^2}{2} $$

<img src="fractal1.png" align="middle" width="160">

¿Cuál es el dominio máximo y el dominio natural de la función $A(x)$?

$\text{Dominio máximo}=\mathbb{R}=(-\infty,\infty)$

$\text{Dominio natural}=(0,\infty)$

* ***Ejemplo:*** El precio de una camioneta nueva es de $\$240,000$ y su valor se deprecia $5\%$ de su valor original al año. 

**1. Escribir un modelo de su valor, $V$, en función de los años transcurridos, $t$.**

Si pasa un año el valor de la camioneta dismuirá en $5\%$.

¿Cuánto es $5\%$ de $\$240000$?

$$ \$240,000\cdot\dfrac{5}{100}=\$12000 $$

In [1]:
240000*5/100

12000.0

Entonces el valor baja a

$$\$240000-\$12000=\$228000$$

In [2]:
240000-12000

228000

Ahora la camioneta tiene el valor de $\$228000$.

Otro año después la camioneta disminuye su valor otros $\$12000$.

In [3]:
228000-12000

216000

Ahora (dos años después) la camioneta vale $\$216000$.

**Paréntesis informativo:**

(Aquí la regué yo :P ; el problema indica que llamemos al valor de la camioneta, $V$, en función de los años transcurridos, $t$. En vez de escribir la función así

$$ f(x)=ax+b $$

tenemos que escribir

$$ V(t)=at+b $$

) **Fin del paréntesis informativo**

Nos gustaría expresar este comportamiento como una función:

$$ V(t)=at+b $$

$V$ representa el valor de la camioneta en función de los años transcurridos, $t$, desde que salió de la agencia.

Pensando la función linela como una recta, $a$ representaría la pendiente de dicha recta. Como con cada año que pasa, el valor baja $\$12000$ podemos pensar que 

$$ a=-12000 $$

Si tenemos dudas con esto podemos pensar en la gráfica del valor de la camioneta (ordenadas) contra el tiempo transcurrido (abscisas). Tenemos las coordenadas:

* $(0,240000)$
* $(1,228000)$

La pendiente para estas coordenadas es

$$ a=\dfrac{228000-240000}{1-0}=-12000 $$

Ahora pensemos qué pasa cuando evaluamos la función en cero:

$$ V(0)=a(0)+b=b $$

Esto respresenta el valor de la camioneta cuando han transcurrido $0$ años:

$$ b=240000 $$

La función nos queda:

$$ V(t)=-12000t+240000 $$

El dominio máximo de está función es $(-\infty,\infty)$.

El rango para este dominio máximo de está función es $(-\infty,\infty)$.

Para el dominio natural nos conviene preguntar: "¿cuál es el mínimo tiempo transcurrido que podemos tener?" La respuesta es simple: $t=0$. Y sabemos que $V(0)=240000$. Este valor de la camioneta es claramente el máximo valor que puede llegar a tener.

"¿Cuál es el mínimo valor que la camioneta puede llegar a tener?" Cero. "¿Cuánto tiempo tendrá que pasar para que la camioneta baje su valor a cero?"

Resolvemos

$$ 0=-12000t+240000 $$

$$ 12000t=240000 $$

$$ t=\dfrac{240000}{12000}=20 $$

In [1]:
240000/12000

20.0

El dominio natural de está función es $[0,20]$.

El rango para este dominio máximo de está función es $[0,240000]$

Para corroborar que nuestro dominio natural y el rango asociado con éste son correctos nos conviene hacer la gráfica:

<img src="grafica_valor.png" align="middle" width="1000">

**2. Si se compró en el 2001, ¿cuál será su valor presente (2020)?**

Habrían transcurrido $19$ años.

$$ f(19)=-12000(19)+240000=12000 $$

In [4]:
-12000*(19)+240000

12000

La camioneta en el $2020$ tiene el valor de $\$12000$.

**3. Si el dueño la quiere vender y la ofrece en $\$120000$, ¿cuántos años han pasado desde su compra?**

$$ y=-12000x+240000 $$

Como quiero que la camioneta valga $120000.

$$ 120000=-12000x+240000 $$

$$ 120000-240000=-12000x $$

$$ -120000=-12000x $$

$$ \dfrac{-120000}{-12000}=x $$

$$ 10=x $$

### Situación: Cantidad de combustible en una moto

El rendimiento de un vehículo expresado como la cantidad de litros de gasolina que
consume por kilómetro es una medida útil para saber el gasto necesario derivado
del uso de éste. Mientras mayor es el rendimiento se obtiene un ahorro mayor de
combustible, en consecuencia, su uso resulta menos costoso y puede tener un
impacto menor en el ambiente. En general, las motocicletas presentan un
rendimiento mayor en relación con un automóvil cuando se trata de transportar a
uno o dos pasajeros. Existe un modelo de motocicleta con un tanque con capacidad
de 15 litros, que cuenta con un rendimiento de 24 kilómetros por litro.

Se desea conocer la cantidad de gasolina en el tanque, a partir de la distancia en
kilómetros recorrida, indicada por la variable $k$, al iniciar un viaje con tanque lleno y sin recargar éste.


**1. A partir de la información de la cantidad de gasolina en el tanque y los kilómetros recorridos mostrada en la siguiente tabla. ¿Cuál es la función que relaciona ambas variables?**


| Kilómetros | Cantidad de gasolina consumida | Cantidad de gasolina en el tanque |
|------------|--------------------------------|-----------------------------------|
| $ 0 $      | $ 0 $                          | $ 15 $                            |
| $ 12 $     | $ 0.5 $                        | $ 14.5 $                          |
| $ 30.6 $   | $ 1.275 $                      | $ 13.725 $                        |
| $ 60 $     | $ 2.5 $                        | $ 12.5 $                          |
| $ 250.2 $  | $ 10.425 $                     | $ 4.575 $                         |
| $ 360 $    | $ 15 $                         | $ 0 $                             |

Queremos tener la función como una función lineal:

$$ G(k)=mk+b $$

La función $G$ me representa la gasolina que me queda en el tanque tras haber recorrido $k$ kilómetors

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ m=\dfrac{14.5-15}{12-0} $$

$$ =\dfrac{-0.5}{12} $$

$$ =-\dfrac{1}{24} $$

$$ G(0)=m(0)+b $$

$b$ es la cantidad de gasolina que queda en el tanque cuando hemos recorrido 0 km.

$$ G(0)=m(0)+b=b=15 $$

La función toma la forma

$$ G(k)=-\dfrac{1}{24}k+15 $$

A) $G(k)=15+\dfrac{1}{24}k$

B) $G(k)=15+24k$

C) $G(k)=15-\dfrac{1}{24}k$

D) $G(k)=15-24k$

**Solución:** C)

**2. Es la cantidad en litros que quedan en el tanque cuando la moto ha recorrido 48
kilómetros:**

La cantidad de gasolina que me queda en el tanque cuando la moto ha recorrido $k$ kilómetros

$$ G(48)=-\dfrac{1}{24}(48)+15=13 $$

A) $17$

B) $2$

C) $13$

D) $6$

**Solución:** C)

**3. ¿Cuántos kilómetros ha recorrido la motocicleta si hay 7 litros de gasolina restantes en el tanque?**


$$ 7=-\dfrac{1}{24}k+15 $$

$$ 7-15=-\dfrac{1}{24}k $$

$$ -8=-\dfrac{1}{24}k $$

$$ \dfrac{-8}{-\dfrac{1}{24}}=k $$

In [5]:
8/(1/24)

192.0

A) $14.9$

B) $360$

C) $192$

D) $353$

**Solución:** C)

**4. ¿Cuál es el máximo dominio de la función?**

$$ (-\infty,\infty) $$

**5. ¿Cuál es el dominio natural de la función?**

$$[0,360]$$

**6. ¿Cuál es el rango para el máximo dominio de la función?**

$$ (-\infty,\infty) $$

**6. ¿Cuál es el rango para el dominio natural de la función?**

$$[0,15]$$

### Modelación de funciones

* ***Ejemplo:*** Un triángulo isósceles que tiene 10 cm de perímetro. Escribe su área en un función de su base.

* Datos:
    * Perímetro $P=10\ \rm{cm}$

* Fórmulas: 
    * $A=\dfrac{b\cdot h}{2}$
    * $P= d+d+b=2d+b$
    * Teorema de Pitágoras: $ h^2+\left(\dfrac{b}{2}\right)^2=d^2 $

* Sustitución

$$10=2d+b$$

Vamos a despejar aquella variable que no sea la variable independiente

Entonces vamos a despejar la diagonal.

$$10-b=2d$$

$$\dfrac{10-b}{2}=d$$

Sustituimos en otra de las relaciones donde aparezca la variable despejada, la variable independiente y otra variable que no conozcamos.

$$ h^2+\left(\dfrac{b}{2}\right)^2=\left(\dfrac{10-b}{2}\right)^2 $$

Vamos a despejar aquella variable que no sea la variable independiente

$$ h^2=\left(\dfrac{10-b}{2}\right)^2-\left(\dfrac{b}{2}\right)^2 $$

$$ h=\sqrt{\left(\dfrac{10-b}{2}\right)^2-\left(\dfrac{b}{2}\right)^2} $$

Sustituimos esto en el área:

$$A(b)=\dfrac{b}{2}\cdot\sqrt{\left(\dfrac{10-b}{2}\right)^2-\left(\dfrac{b}{2}\right)^2}$$

https://www.desmos.com/calculator/qwan31db49

## Logaritmos

Primero hablemos de potencias. Si tenemos

$$3^4=81$$

¿Qué es un logaritmo?

Es la respuesta a la pregunta de cuántas veces tengo que multiplicar a un número por sí mismo para lograr un resultado.

El resultado $ \log_3(81) $ es el número de veces que tengo que multiplicar la base por sí misma para obtener el valor dentro del logaritmo.

$$ \log_3(81)=4 $$

* ***Ejemplo:***

$$ \log_2(8)=3 $$

### Leyes de los logaritmos

* $\log_a(1)=0$

* ***Ejemplo:***

$$ \log_7(1)=0 $$

* $\log_a(a)=1$

* ***Ejemplo:***

$$ \log_6(6)=1 $$

* $\log_a(x)+\log_a(y)=\log_a(xy)$

$$\log_6(100)$$

$$\log_6(2)+\log_6(50)=$$

* ***Ejemplo:***

* $\log_a(x)-\log_a(y)=\log_a\left(\dfrac{x}{y}\right)$

* $\log_a(x^n)=n\log_a(x)$

* ***Ejemplo:***

$$\log_2(27)=\log_2(3^3)=3\log_2(3)$$

* $\log_a(b)=\dfrac{\log_c(b)}{\log_c(a)}$

Existe un logaritmo muy especial que se conoce como el logaritmo natural.

La base de este logaritmo es $e=2.718281828459045...$

In [7]:
exp(1)

2.718281828459045

$$\log_{10}(10567)=$$

In [9]:
log10(10567)

4.023951707423388

In [10]:
10^(4.023951707423388)

10566.999999999993